src/app/in-memory-data.service.ts

```diff
@@ -1,7 +1,7 @@
 import { InMemoryDbService } from 'angular2-in-memory-web-api';
 export class InMemoryDataService implements InMemoryDbService {
   createDb() {
-    let workItems = [
+    let workitems = [
```

Comment:
That was a tough one to find. It seems that the name of the variable is actually reflected as the URL part. As the real api uses "../workitems/", this has also to be consistent here.